scio-core/src/main/scala/com/spotify/scio/values/DoubleSCollectionFunctions.scala

```diff
@@ -39,22 +39,26 @@ class DoubleSCollectionFunctions(self: SCollection[Double]) {
   // def sum: SCollection[Double] = this.stats().map(_.sum)
 
   /** Compute the standard deviation of this SCollection's elements. */
-  def stdev: SCollection[Double] = self.transform(_.stats.map(_.stdev))
+  def stdev: SCollection[Double] = self.transform(_.stats.withName("StdDev").map(_.stdev))
```

Comment:
Yeah maybe we can remove tests for most types derived `SCollection` and assert the part before `/`. Leave the inner ones alone for now and fix when it becomes a problem.